caluma/caluma_form/utils.py

```diff
@@ -1,7 +1,79 @@
+from django.db.models import Prefetch
+
 from caluma.caluma_form import models, structure
 from caluma.caluma_form.jexl import QuestionJexl
 
 
+def build_document_prefetch_statements(prefix="", prefetch_options=False):
+    """Build needed prefetch statements to performantly fetch a document.
+
+    This is needed to reduce the query count when almost all the form data
+    is needed for a given document, e.g. when recalculating calculated
+    answers.
```

Comment:
Docstring needs an example I think.

Also I feel like this is not quite the right abstraction level just yet. Callers still have quite a bit of work to do to get it right

Reply:
I tried to add an example and changed the callsite